package.json

```diff
@@ -57,17 +57,17 @@
     "tap": "^14.0.0"
   },
   "scripts": {
-    "unit": "tap --100 --coverage --coverage-report=text ./tests/test_*.js",
-    "pretest": "npm run -s lint",
-    "test": "npm run -s unit",
-    "posttest": "npm run -s prettier:check",
-    "coverage": "tap --coverage-report=html && open coverage/lcov-report/index.html",
-    "mocha": "nyc mocha $(grep -lr '^\\s*it(' tests)",
-    "lint": "eslint \"**/*.js\"",
-    "prettier": "prettier --write \"**/*.@(js|json|md|ts|yml)\"",
-    "prettier:check": "prettier --check \"**/*.@(js|json|md|ts|yml)\"",
-    "dtslint": "dtslint types",
-    "semantic-release": "semantic-release"
+    "format:fix": "prettier --write '**/*.{js,json,md,ts,yml,yaml}'",
+    "format": "prettier --check '**/*.{js,json,md,ts,yml,yaml}'",
+    "lint": "run-p lint:js lint:ts",
+    "lint:js": "eslint --cache --cache-location './.cache/eslint' '**/*.js'",
+    "lint:js:fix": "eslint --cache --cache-location './.cache/eslint' --fix '**/*.js'",
+    "lint:ts": "dtslint types",
+    "semantic-release": "semantic-release",
+    "test": "run-p test:mocha test:tap",
+    "test:coverage": "tap --coverage-report=html && open coverage/lcov-report/index.html",
+    "test:mocha": "nyc mocha $(grep -lr '^\\s*it(' tests)",
+    "test:tap": "tap --100 --coverage --coverage-report=text ./tests/test_*.js"
```

Comment:
These I'm not 100% sure about. It seems like you guys are switching from Tap to Mocha?